### PR TITLE
 Config and run stylelint correctly.

### PR DESCRIPTION
**Pull request by beausmith**
Figured out how to use stylelint for both css and css-in-js files.

### CODE REVIEW

File: cypress/tests/scroll-to-contest.js [hunk 1]
```diff
@@ -1,9 +1,4 @@
-import electionSample from '../../src/data/electionSample'
-
-const clickThoughPages = [
-  ...electionSample.contests.slice(0, 20),
-  { id: 'pre-review-page' },
-]
+const clickThoughPages = new Array(21) // number of contests + pre-review screen for activation code 'VX.23.12'
```
Comment (benadida) [inline cypress/tests/scroll-to-contest.js]: sneaking in an unrelated change I see??? ;) It's fine.

File: src/pages/PrintPage.tsx [hunk 2]
```diff
@@ -214,7 +214,8 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
               <Header>
                 <div
                   className="seal"
-                  dangerouslySetInnerHTML={{ __html: seal }}
+                  // TODO: Sanitize the SVG content: https://github.com/votingworks/bmd/issues/99
```
Comment (benadida) [inline src/pages/PrintPage.tsx]: 👍

File: cypress/tests/scroll-to-contest.js (hunk 1 above)
Comment (beausmith) [reply cypress/tests/scroll-to-contest.js]: No sneaky business. 😄 Seems unrelated, but I was getting an ESLint error saying there was an issue importing the `electionSample.json` file (even though Cypress runs fine). While I was troubleshooting it, I realized that since ballot style support was added, this test no longer relies upon the data in the file to determine the length of contests… so why not just manually specify the length another way and remove the import?
